root_VS2012/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Views/Shared/_CrudSampleLayout.cshtml

```diff
@@ -9,13 +9,20 @@
     @Scripts.Render("~/bundles/modernizr")
     <script src="~/Scripts/jquery-1.8.2.min.js"></script>
     <script src="~/Scripts/jquery-ui-1.9.2.min.js"></script>
+    <script src="~/Framework/Js/common.js"></script>
     <script src="~/Scripts/jquery.unobtrusive-ajax.min.js"></script>
-    <link href="~/Content/themes/base/jquery-ui.css" rel="stylesheet" type="text/css" />   
+    <link href="~/Content/themes/base/jquery-ui.css" rel="stylesheet" type="text/css" />
     <link href="~/Content/themes/base/jquery.ui.all.css" rel="stylesheet" type="text/css" />
     <link href="~/Content/themes/base/smoothness/jquery-ui-1.10.2.custom.css" rel="stylesheet" type="text/css" />
     @RenderSection("scripts", required: false)
+    <script type="text/javascript">
+        URL = '@Url.Action("Index", "Ping")';
+        $("form").live("submit", function (event) {
+            Fx_OnSubmit();
```

Comment:
@yusukemaegawa 

I think that there is no need to set the method of the submit event.
And, I think it should be treated with GET method instead of POST method.

https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/C%23/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js#L67

Please tell me, what do you think.